commands/git/git-edit.md

```diff
@@ -0,0 +1,59 @@
+### Edit a file
```

Comment:
I think we should change this title to something else, as the commands is not related to editing a file.
How about changing it to `Commit a change` and file name to `git-commit`

Any thoughts?

Reply:
How about Stage and Commit?

Reply:
yes. this sounds better and also makes sense as per the content of this file